transformer/gen3_transformer.py

```diff
@@ -8,13 +8,28 @@
 
 import argparse
 
+# Welcome to the gen3 transformer. Here is an overview of how the gen3 data is parsed and transformed.
```

Comment:
`gen3` and `sheepdog` should be capitalized. Otherwise, looks good!

Reply:
It looks like sheepdog doesn't capitalize anywhere. https://github.com/david4096/sheepdog-exporter